src/crud.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/no-cycle, object-curly-newline */
+import { items, AddItem, setStored, displayList, checkList } from './index.js';
+
+const addCrud = () => {
+  const addEnter = document.querySelector('.add-enter');
+  addEnter.addEventListener('keyup', ({ key }) => {
+    if (key === 'Enter' && addEnter.value !== '') {
+      const newEntry = new AddItem();
+      newEntry.description = addEnter.value;
+      newEntry.index = items.length;
+      items.push(newEntry);
+      addEnter.value = '';
+      checkList.innerHTML = '';
+      setStored();
+      displayList();
+    }
+  });
+};
+
+const removeCrud = (trashIcon) => {
```

Comment:
- [ ] When I try to remove a task from the list via the trash icon, sometimes, other tasks get removed aswell. I think this is not the intended behavior. Please, let's review the functions related and try to fix this bug.

![todo_list_deletes_more_items](https://user-images.githubusercontent.com/79114233/145509110-5450712a-709f-4fd1-8649-e4224067bc8f.gif)